### ISSUE TITLE
add nice "report a problem with the web site" link on each page

### ISSUE THREAD

**Issue by tkimnguyen** [plone/ploneorg.core#137]
...ideally one of those floating things that comes in from the side? that links to here (https://github.com/plone/ploneorg.core/issues/)

**Comment by tkimnguyen** [comment]
Bah! of course we already had that in our design, in #111.

**Comment by polyester** [comment]
yeah, but that was way too big. So a floading 'feedback' like now on http://docs.plone.org is the less intrusive one. I'll ask Sven how that's done.
